pywbem/_server.py

```diff
@@ -776,6 +806,14 @@ def get_central_instances(self, profile_path, central_class=None,
               :class:`~pywbem.CIMInstanceName`.
         """  # noqa: E501
         # pylint: enable=line-too-long
+
+        if reference_direction != ('dmtf', 'snia'):
+            ValueError('Direction parameter must be "dmtf" or "snia". '
+                       'Parameter "%s" is invalid.' % reference_direction)
```

Comment:
Name the parameter correctly, it is named "reference_direction".
Also, use new style formatting.
Third, we have the `_format()` function that makes sure unicode formatting arguments are presented correctly.

Suggestion:
```
ValueError(
    _format("The reference_direction parameter must be 'dmtf' or 'snia', "
            "but is: {0!A}", reference_direction))
```

Reply:
fixed